scripts/beegfsmd.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+#
+disk_type=$1
+node_type=$2
+pools=$3
+MGMT_HOSTNAME=$4
+#
+BEEGFS_DISK=/mnt/beegfs
+BEEGFS_METADATA=${BEEGFS_DISK}/meta
+#
+yum install -y beegfs-meta
+sed -i 's|^storeMetaDirectory.*|storeMetaDirectory = '$BEEGFS_METADATA'|g' /etc/beegfs/beegfs-meta.conf
+sed -i 's/^sysMgmtdHost.*/sysMgmtdHost = '$MGMT_HOSTNAME'/g' /etc/beegfs/beegfs-meta.conf
+#
+sed -i 's/^connMaxInternodeNum.*/connMaxInternodeNum = 800/g' /etc/beegfs/beegfs-meta.conf
+sed -i 's/^tuneNumWorkers.*/tuneNumWorkers = 128/g' /etc/beegfs/beegfs-meta.conf
+#
+setup_data_disks()
+{
+    mountPoint="$1"
+    filesystem="$2"
+    devices="$3"
+    raidDevice="$4"
+    createdPartitions=""
+    numdevices=`echo $devices | wc -w`
+    if [ $numdevices -gt 1 ]
```

Comment:
The code to create a raid is duplicated here (well, some of it)

Reply:
yes agreed, this should be improved, but for now would like to merge this version and make improvments going forward.